tests/Unit/IO/Test_VolumeData.py

```diff
@@ -17,19 +17,86 @@ def setUp(self):
         # these will be deleted and recreated for each test as needed.
```

Comment:
Now that I look at this I don't think we still need the `file_name_w` and `file_name_r`, you can probably just get rid of `file_name_r`, and change all of the files to just use `self.file_name = os.path.join(Informer.unit_test_path(),
                                        "IO/pythontest.h5")`